### PR TITLE
Prac 06 feedback

### PR DESCRIPTION
**Pull request by Eggnt**
please review @cheyenneIT22 
thank you for any feedback

### CODE REVIEW

File: prac_06/used_cars.py [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+    # print(f"Car has fuel: {my_car.fuel}")
+    # print(my_car)
+    limo = Car("Limousine", 100)
+    limo.add_fuel(20)
+    print(f"{limo.name} has fuel: {limo.fuel}")
+    limo.drive(115)
+    print(f"{limo.name} has fuel: {limo.fuel}")
+    print(limo)
+
+
```
Comment (cheyenneIT22) [inline prac_06/used_cars.py:23]: Very great code in both the classes and the client code. good job

File: prac_06/guitars.py [hunk 2]
```diff
@@ -3,8 +3,6 @@
 
 def main():
     guitars = []
-    guitars.append(Guitar("Gibson L-5 CES", 1922, 16035.40))
-    guitars.append(Guitar("Line 6 JTV-59", 2010, 1512.9))
     print("My guitars!")
     name = str(input("Name: "))
     while name != "":
```
Comment (cheyenneIT22) [inline prac_06/guitars.py:8]: was this code suppose to be apart of the main code?

**Review by cheyenneIT22 — COMMENTED** [review]
All your code looks Great! Good job with linking the classes to the client code as well. :)

File: prac_06/guitars.py (hunk 2 above)
Comment (Eggnt) [reply prac_06/guitars.py:8]: I may have accidently left in in.

Comment (cheyenneIT22) [reply prac_06/guitars.py:8]: oh that is okay lol.

**Pull request merged**